docs/_manage/account-and-team/add-team.md

```diff
@@ -8,33 +8,63 @@ redirect_from:
+## Inviting Team Members
 
-> **Note**: Admins can invite both admins and collaborators while collaborators can only invite collaborators.
+To add team members to your integration, follow these steps:
 
-7. Add a **note** to the invitation. This information will be included in the invitation email.
-8. Click the **Send Invite** button.
+1. Log into the [Developer Platform](https://zapier.com/app/developer)
+2. Choose your integration
+3. In the "Manage" section on the left sidebar, click "Manage Team”
+4. Click “Invite Team Member”
+5. Enter the email address of the team member that you want to invite
+6. Select the role that you wish to assign to the team member
+7. A note will be included in the invitation email. You can either use the default message provided, or customize it with your own text
+8. Click “Send Invite”
 
-When you do this, an invitation email would be sent to the invitee, asking them to accept the invitation.
+> **Note**: Admins can invite both Admins and Collaborators, while Collaborators can only invite Collaborators.
 
-Once an invitee accepts the invite, they can access the integration through their own Zapier account. If the invitee does not have a Zapier account, accepting the invite would prompt them to sign up one first before they can access the integration.
+Once you click "Send Invite", an email invitation will be sent to the invitee, requesting their acceptance. If the invitee does not already have a Zapier account associated with the invited email, they will need to create an account first. Upon accepting the invitation, they will gain access to the integration through their Zapier account on the Developer Platform.
 
-## View and remove team members
+## Self-Serve Collaborator Access
 
-You can view a list of invited team members (both those that have accepted the invites and those with pending invites) from the **Manage Team** page. An admin can also remove other team members from this page. However, please note that if you are removed as an Admin, you will only be able to regain Admin access if invited by an existing Admin.
\ No newline at end of file
+Depending on your [integration settings](https://cdn.zappy.app/b2637f3ad910c36c4e8c8224c349beee.png), users have the option to self-serve and join an integration team as Collaborators. This feature employs domain-name verification and is managed by integration Admins. Currently, this feature is only available for integrations listed in [Zapier's directory](https://zapier.com/apps).
+
+### Enabling or Disabling Self-Serve Collaborator Access
+
+To toggle the “Self-serve join as Collaborator” feature, please follow these steps:
+1. Log into the [Developer Platform](https://zapier.com/app/developer)
+2. Choose your integration
+3. In the "Manage" section on the left sidebar, click "Manage Team"
+4. Click “Settings”
+5. Click the toggle to enable or disable self-serve join as Collaborator
+
+### Managing Eligible Email Domains for Self-Serve Collaborator Access
```

Comment:
Manage eligible email domains for self-serve collaborator access

Reply:
Fixed, but I kept the capital "C" for Collaborator due to how it's being used as a verb. Collaborator is capitalized everywhere else, shouldn't it also be capitalized here? Submitted new push.